### PR TITLE
Added New Keyboard

### PR DESCRIPTION
**Pull request by MATTMCCA**
Added Heliotrope to the QMK Keyboard directory.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x ] Keyboard (addition or update)
- [ x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ x ] I have updated the documentation accordingly.
- [ x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/heliotrope/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# heliotrope
+
+![heliotrope](https://github.com/MATTMCCA/Heliotrope/blob/main/Case/test_fit.jpeg)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/heliotrope/readme.md:5]: Should be filled in or removed

File: keyboards/heliotrope/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# This file intentionally left blank
```
Comment (zvecr) [inline keyboards/heliotrope/keymaps/via/rules.mk:1]:
```suggestion
```

File: keyboards/heliotrope/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
+﻿#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/heliotrope/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2022 MATTMCCA (@MATTMCCA)
// SPDX-License-Identifier: GPL-2.0-or-later

﻿#include QMK_KEYBOARD_H
```

File: keyboards/heliotrope/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,49 @@
+﻿#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/heliotrope/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2022 MATTMCCA (@MATTMCCA)
// SPDX-License-Identifier: GPL-2.0-or-later

﻿#include QMK_KEYBOARD_H
```

File: keyboards/heliotrope/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+	),
+
+	[3] = LAYOUT_60_ansi(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	)
+
```
Comment (zvecr) [inline keyboards/heliotrope/keymaps/default/keymap.c:48]:
```suggestion
	)
```

File: keyboards/heliotrope/config.h [hunk 6]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 MATTMCCA (@MATTMCCA)
```
Comment (zvecr) [inline keyboards/heliotrope/config.h:1]: This file can be removed

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/heliotrope/config.h (hunk 6 above)
Comment (waffle87) [inline keyboards/heliotrope/config.h:1]: This file should be deleted.

File: keyboards/heliotrope/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# heliotrope
+
+![heliotrope](https://github.com/MATTMCCA/Heliotrope/blob/main/Case/test_fit.jpeg)
```
Comment (waffle87) [inline keyboards/heliotrope/readme.md:3]: This image should be rehosted on Imgur.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/heliotrope/readme.md (hunk 1 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/readme.md]: removed per request

File: keyboards/heliotrope/keymaps/default/keymap.c (hunk 3 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/keymaps/default/keymap.c]: changed, added copyright as requested.

File: keyboards/heliotrope/keymaps/via/keymap.c (hunk 4 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/keymaps/via/keymap.c]: changed

File: keyboards/heliotrope/keymaps/default/keymap.c (hunk 5 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/keymaps/default/keymap.c]: removed 3 trans layers as requested.

File: keyboards/heliotrope/config.h (hunk 6 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/config.h]: file was nuked as requested.

File: keyboards/heliotrope/keymaps/via/rules.mk (hunk 2 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/keymaps/via/rules.mk:1]: removed as requested.

**Review by zvecr — CHANGES_REQUESTED** [review]
`keyboards/heliotrope/rules.mk` still needs to have the `# This file intentionally left blank` line

**Comment by MATTMCCA** [comment]
Thank you for your patience with me.

File: keyboards/heliotrope/info.json [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+        "nkro": true,
+	    "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 61,
+        "pin": "A4",
+        "hue_steps": 10,
+		"max_brightness": 255,
+        	"animations": {
+			"all": true
```
Comment (drashna) [inline keyboards/heliotrope/info.json:22]: Could you explicitly define which animations are enabled? 

```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/heliotrope/info.json (hunk 8 above)
Comment (MATTMCCA) [reply keyboards/heliotrope/info.json]: changed as requested.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**